spec/features/courses/details_spec.rb

```diff
@@ -140,7 +140,7 @@
       course_details_page.load_with_course(course)
       expect(course_details_page).to have_entry_requirements
       expect(course_details_page.entry_requirements).to have_content("Maths GCSE: Taking")
-      expect(course_details_page.entry_requirements).to have_content("Science GCSE: Equivalence test")
+      expect(course_details_page.entry_requirements).to have_content("Science GCSE: Equivalency test")
```

Comment:
Why in this PR?

Reply:
Paul G asked for it so done as a sperate commit